sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala

```diff
@@ -1520,7 +1520,7 @@ abstract class RoundBase(child: Expression, scale: Expression,
         if (_scale >= 0) {
           s"""
             ${ev.value} = ${ce.value}.toPrecision(${ce.value}.precision(), $s,
-            Decimal.$modeStr(), true, "");
+            Decimal.$modeStr(), true, scala.None$$.MODULE$$);
```

Comment:
shall we provide a default parameter value `None` to avoid this?

Reply:
It is Java code. Do Scala's default params work in Java? Since it is used only once, not sure it worth to do so.

Reply:
ok let's leave it then